Resources/Prototypes/Entities/Mobs/Species/diona.yml

```
@@ -112,6 +112,10 @@
   - type: FootPrints # WD EDIT
     leftBarePrint: "footprint-left-bare-diona"
     rightBarePrint: "footprint-right-bare-diona"
+  - type: StepTriggerImmune # Goobstation
```

yamllint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)